docs/source/conf.py

```diff
@@ -180,3 +180,8 @@ def setup(app: Sphinx):
 suppress_warnings = [
     "autosummary.import_cycle",
 ]
+
+# -----------nbsphinx extension ----------
+nbsphinx_execute = "never"  # always
```

Comment:
If you never run the notebooks, do the below the configurations have any effect?

Reply:
I found that there is a workflow run for running the notebook, this is for building the docs.

Reply:
If I am not wrong, `nbsphinx_execute = "never"` will ensure that RTD will not execute the example notebooks while building the docs (ref. https://nbsphinx.readthedocs.io/en/0.9.5/never-execute.html). If that is the case, whether that execution need to ignore errors or not (`nbsphinx_allow_errors = False`) or how long execution takes (`nbsphinx_timeout = 600`) should not matter.

Does that make sense? Let me know if I am missing something, or misunedrstanding the purpose of these configurations.

Reply:
You're right, these extra configurations are meaningless.
should we keep the `nbsphinx_execute = "never"`? as we run notebooks in this workflow to test if they don't fail.https://github.com/sktime/pytorch-forecasting/blob/e08f4e6d23737c36949d704215ccbbbaa84fc1b6/.github/workflows/test.yml#L17,

Reply:
I believe `never` is fine, these executions increase RTD time significantly. Let's skip in `sktime` too for that matter, if @fkiraly agree as well (I have seen too many timeouts recently).